hubspot/plugin.py

```diff
@@ -135,57 +478,61 @@ def create_contacts(contacts: List[HubSpotContact]) -> List[str]:
 
 
 @dataclass
-class HubSpotContactUpdate:
-    """The data required to update a HubSpot contact.
+class ContactDefaultPropertyUpdate:
+    """The data required to update a contact's default properties."""
 
-    Attributes:
-        id: The HubSpot ID of the contact to update.
-        updated_properties: A dictionary of property names and values to update for the
-            contact. Only properties that are to be updated need to be included.
+    contact_id: str
+    updated_properties: List[Tuple[HubSpotContactDefaultPropertyName, str]]
 
-    """
 
-    id: str
-    updated_properties: Dict[str, str]
+@dataclass
+class ContactCustomPropertyUpdate:
+    """The data required to update a contact's custom properties."""
+
+    contact_id: str
+    updated_properties: List[Tuple[HubSpotContactCustomPropertyName, str]]
 
 
-def update_contacts(contacts: List[HubSpotContactUpdate]) -> List[str]:
+def update_contacts(
+    default_properties: List[ContactDefaultPropertyUpdate],
```

Comment:
Sequence instead of List for input types (to be consistent with the other functions)?